engine/src/flutter/shell/platform/android/io/flutter/embedding/engine/FlutterEngine.java

```diff
@@ -378,6 +384,7 @@ public FlutterEngine(
 
     this.platformViewsController = platformViewsController;
     this.platformViewsController.onAttachedToJNI();
+    this.platformViewsController2 = platformViewsController2;
```

Comment:
missing call to onAttachedToJNI() here?

Reply:
This method was a no-op so I didn't implement this for pvc2. To make this less confusing I went ahead and deleted the method on pvc1